src/panels/lovelace/entity-rows/hui-media-player-entity-row.js

```diff
@@ -103,22 +103,40 @@ class HuiMediaPlayerEntityRow extends LocalizeMixin(PolymerElement) {
   _computeMediaTitle(stateObj) {
     if (!stateObj || this._isOff(stateObj.state)) return null;
 
+    // Default to guessing a prefix.
```

Comment:
I don't think these comments are necessary. The logic seems pretty straight-forward